OO.Rmd

```diff
@@ -6,55 +6,61 @@ source("common.R")
 library(sloop)
 ```
 
+**GVW:** After going through this doc, base-types.Rmd, and S3.Rmd, my impression is that this material is written for people who've done OO before in some other language and are encountering generic function-style OO in R for the first time.
```

Comment:
Would you mind putting these in html comments? That way I can merge the minor changes and systematically work my way through the biggers thoughts

Reply:
:+1: